sec_certs/certificate/certificate.py

```diff
@@ -52,7 +52,7 @@ def to_json(self, output_path: Union[Path, str]):
     @classmethod
     def from_json(cls, input_path: Union[Path, str]):
         with Path(input_path).open('r') as handle:
-            return json.load(handle, cls=CustomJSONDecoder)
+            return json.load(handle, csl=CustomJSONDecoder)
```

Comment:
@GeorgeFI Don't the tests fail because of this? The line should be `return json.load(handle, cls=CustomJSONDecoder)`. Otherwise I can't see why the tests are failing (everything else looks good to me). Provided that I have more insight, I'll try to run your branch this evening to see the problem.

Reply:
@GeorgeFI I took a look at the test and figured that the problem was already previously present in the code (and it was me who introduced it), sorry for that! The problem is that each subclass of `ComplexSerializableType` must have unique name. But when I introduced CPE matching to FIPS dataset, I created an attribute `Heuristics` which was already a `ComplexSerializableType` of CommonCriteriaDataset. 

I hotfixed the problem by renaming `Heuristics` types to `FIPSHeuristics`, `CCHeuristics` respectively. The longterm fix is probably include whole module path for serialization, e.g., `sec-certs.certificate.fips.heuristics` instead of `heuristics`. 

There still was one test not passing, caused by a typo in the template `toy_dataset.json`. I fixed that too 👍.